### DISCUSSION TITLE
Lanbon Burnout - Hardware Failure?

### DISCUSSION THREAD

**Discussion by erwin** [HASwitchPlate/openHASP#370] [General]
I started my OpenHASP adventure with 5 x Lanbon L8s about six months ago. Mostly it's been fantastic.

About 2 moths ago, one of two bedside lanbon's died. The screen still works fine, but the underlying switch failed.

I contacted Lanbon and purchased a replacement switch (not screen - just the electrical switch component) and have been waiting for our electrician to have time to install it.

Yet before the install, now my other bedside lanbon has died.

Originally my house did not have a neutral wire, so I had neutral wires added to all switches.

I'm wondering if two switches that are adjacent on the same line "burning out" (that's speculation on my part - but why else would only those two have an issue).

The other thing unusual about those two switches is that are are completely virtual --- I'm not using them to control any other devices, only to interact with other things via Home Assistant MQTT.

I have one more "virtual switch" that's still working, and my two plates that also control one or more light switches is still working perfectly.

I'm open to doing whatever needs to be done to get it working reliably...  Maybe there's some kind of voltage test over time I can do to see if I'm getting spikes on that circuit?

I have almost no experience in electrical or switches outside of my openhasp experiments, so very grateful for any ideas!

**Comment by nagyrobi** [comment]
What model?
US, EU, AU?
